package.json

```diff
@@ -0,0 +1,33 @@
+{
+	"name": "photos-network-frontend",
+	"description": "Photos.network front-end",
+	"version": "0.0.1",
+	"author": "Tom",
```

Comment:
change author to the core team or all developers e.g. Photos network developers

Reply:
can it be just `Photos.network`?